src/main/java/com/google/devtools/build/lib/analysis/test/TestConfiguration.java

```diff
@@ -281,6 +281,14 @@ public static class TestOptions extends FragmentOptions {
         help = "If true, undeclared test outputs will be archived in a zip file.")
     public boolean zipUndeclaredTestOutputs;
 
+    @Option(
+        name = "use_target_platform_for_tests",
+        defaultValue = "false",
+        documentationCategory = OptionDocumentationCategory.EXECUTION_STRATEGY,
+        effectTags = {OptionEffectTag.EXECUTION},
+        help = "If true, then Bazel will run coverage postprocessing for test in a new spawn.")
```

Comment:
This help message is entirely wrong. :)

Reply:
Whops, done.  Enough people have worked on this commit that something got lost in translation :)